main.py

```diff
@@ -74,15 +80,47 @@ def log(s):
   print('[' + str(datetime.now().strftime('%Y-%m-%dT%H:%M:%S')) + '] ' + s)
 
 
+def load_memory(memory_path, disable_bzip):
+  if disable_bzip:
+    with open(memory_path, 'rb') as pickle_file:
+      return pickle.load(pickle_file)
+
```

Comment:
Please remove line space between `if` and `else` conditions in these functions. Functions and logic are fine.